src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -67,41 +112,71 @@ const SubsidiesSummary = ({
+    if (!hasAssignedCodesOrCodeRequests && (
+      disableExpiryMessagingForLearnerCredit || !subscriptions.showExpirationNotifications
+    )) {
+      // If notifications for both learner credit and subscriptions are disabled, and they are both expired
+      // We do not return a class name
+      if (
+        hasApplicableLearnerCredit && hasActiveLicenseOrLicenseRequest
+          && !subscriptions.showExpirationNotifications
+          && !subscriptions?.subscriptionPlan?.isCurrent
+          && disableExpiryMessagingForLearnerCredit
+          && dayjs(learnerCreditSummaryCardData?.expirationDate).isBefore(dayjs())
+      ) {
+        return undefined;
+      }
+      // If notifications are enabled for one type of subsidy but not the other and they are expired
+      // We also render undefined
+      if (
+        hasActiveLicenseOrLicenseRequest && !hasApplicableLearnerCredit
+          && !subscriptions?.showExpirationNotifications
+          && !subscriptions?.subscriptionPlan?.isCurrent
+      ) {
+        return undefined;
+      }
+      if (
+        hasApplicableLearnerCredit && !hasActiveLicenseOrLicenseRequest
+          && disableExpiryMessagingForLearnerCredit
+          && dayjs(learnerCreditSummaryCardData?.expirationDate).isBefore(dayjs())
+      ) {
+        return undefined;
+      }
+    }
+    // Otherwise, we render the default CSS value
+    return 'pt-0';
+  }, [
+    disableExpiryMessagingForLearnerCredit,
+    hasActiveLicenseOrLicenseRequest,
+    hasApplicableLearnerCredit,
+    hasAssignedCodesOrCodeRequests,
+    learnerCreditSummaryCardData?.expirationDate,
+    subscriptions.showExpirationNotifications,
+    subscriptions?.subscriptionPlan?.isCurrent,
+  ]);
+
   if (!hasAvailableSubsidyOrRequests) {
     return null;
   }
 
-  const searchCoursesCta = (
-    !programProgressPage && !enterpriseCustomer.disableSearch && showSearchCoursesCta && (
-      <Button
-        as={Link}
-        to={`/${enterpriseCustomer.slug}/search`}
-        variant={ctaButtonVariant}
-        block
-      >
-        <FormattedMessage
-          id="enterprise.dashboard.sidebar.subsidy.find.course.button"
-          defaultMessage="Find a course"
-          description="Button text for the find a course button on the enterprise dashboard sidebar."
-        />
-      </Button>
-    )
-  );
-
   return (
-    // TODO: Design debt, don't have cards in a card
-    <SidebarCard
-      cardSectionClassNames="border-0 shadow-none p-0"
-      cardClassNames={classNames('mb-5', { 'col-8 border-0 shadow-none': programProgressPage })}
+    <Card
+      className={classNames('mb-5', { 'col-8 border-0 shadow-none': programProgressPage })}
     >
       <div className={className} data-testid="subsidies-summary">
         {hasActiveLicenseOrLicenseRequest && (
           <SubscriptionSummaryCard
             subscriptionPlan={subscriptions.subscriptionPlan}
+            showExpirationNotifications={subscriptions.showExpirationNotifications}
```

Comment:
[nit/curious] Would it be helpful to reduce the number of props and simply pass `subscriptions={subscriptions}` here versus splitting it out to 2 independent props?